packages/components/src/List/ListComposition/Manager/hooks/useCollectionFilter.hook.test.js

```diff
@@ -139,6 +150,27 @@ describe('useCollectionFilter', () => {
 		expect(filteredCollection[2].firstName).toEqual('Carly');
 	});
 
+	it('should filter with limited column list', () => {
+		// given
+		const wrapper = mount(
+			<FilterComponent collection={collection} initialFilteredColumns={['lastName']} />,
+		);
+
+		let filteredCollection = wrapper.find('#mainChild').prop('filteredCollection');
+		expect(filteredCollection).toBe(collection);
+
+		// when
+		act(() => {
+			wrapper.find('#mainChild').prop('setTextFilter')('l');
```

Comment:
I can be wrong, but for me this test does not test anything. `Silva Madder` is the first item with an `l` in its row.